### PR TITLE
Full audit

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3da109fd5d/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3da109fd5d/how-to/)

**Comment by catlook** [comment]
waiting on final review by Honsou

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e9b548d718/how-to/)

**Comment by catlook** [comment]
Steph, edit please?

File: content/general/comparison-of-dedicated-and-cloud-hosting.md [hunk 1]
```diff
@@ -26,26 +26,27 @@ your business.
...
+    <td>Same options as dedicated, if using RackConnect.</td>
+    <td>Same options as dedicated, if using RackConnect.</td>
   </tr>
   <tr>
     <td><strong>Monitoring</strong></td>
-    <td>RackWatch monitoring is included and is enabled by default for PING service checks. Upon request, six additional service checks and one free URL monitor can be configured.</td>
-    <td>Managed Operations includes several checks and alerts through the Cloud Monitoring service. Support helps you with the creation and configuration of monitoring checks.</td>
-    <td>Monitoring is not preconfigured for Managed Infrastructure, but Cloud Monitoring can be manually set up after server creation. There is no charge for the Cloud Monitoring service.<br><br>Without a third-party monitoring solution or use of the Cloud Monitoring service, you are not notified when individual services on your cloud server stop working. You are notified only if the host server where your cloud server is located is impacted.</td>
+    <td>Rackspace Monitoring is included by default, with ping alarms automatically set to create tickets forRackspace admins.</td>
```
Comment (stephamon) [inline content/general/comparison-of-dedicated-and-cloud-hosting.md]: missing space between "for" and "Rackspace"

File: content/general/comparison-of-dedicated-and-cloud-hosting.md [hunk 2]
```diff
@@ -26,26 +26,27 @@ your business.
...
+    <td>Rackspace Monitoring is included by default. Checks are created, but alarms are optional.  Alarms can be set notify your account and to automatically create tickets for Rackspace admins. </td>
+    <td>Rackspace Monitoring included by default. Checks are created if the optional Rackspace Monitoring Agent is installed on your cloud server, but alarms are optional. Alarms can notify your account, but cannot automatically create tickets for Rackspace admins. </td>
```
Comment (stephamon) [inline content/general/comparison-of-dedicated-and-cloud-hosting.md]: Rackspace Monitoring is included by default.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ebe6761e2f/how-to/)

**Comment by catlook** [comment]
merging self.

**Pull request merged**